dsa-problems/leetcode-problems/0300-0399.md

```diff
@@ -21,7 +21,7 @@ export const problems = [
         "problemName": "301. Remove Invalid Parentheses",
         "difficulty": "Hard",
         "leetCodeLink": "https://leetcode.com/problems/remove-invalid-parentheses/",
-        "solutionLink": "#"
+        "solutionLink": "dsa-solutions/lc-solutions/0300-0399/0301-Remove-Invalid-Parentheses"
```

Comment:
For good practice, you need to start your link with a `/`. For example, replace `dsa-solutions/lc-solutions/0300-0399/0301-Remove-Invalid-Parentheses` with `/dsa-solutions/lc-solutions/0300-0399/0301-Remove-Invalid-Parentheses`.

Reply:
Sure sir, will update the PR immediately.

> For good practice, you need to start your link with a `/`. For example, replace `dsa-solutions/lc-solutions/0300-0399/0301-Remove-Invalid-Parentheses` with `/dsa-solutions/lc-solutions/0300-0399/0301-Remove-Invalid-Parentheses`.

Reply:
done